librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -996,6 +996,157 @@ commands:
           - name: number
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+  - name: ag
+    summary: Analysis graph commands
+    details:
+      - name: "Formats"
+        entries:
+          - text: "<blank>"
+            comment: "Ascii art"
+          - text: "*"
+            comment: "rizin commands"
+          - text: "d"
+            comment: "Graphviz dot"
+          - text: "g"
+            comment: "Graph Modelling Language (gml)"
+          - text: "j"
+            comment: "json ('J' for formatted disassembly)"
+          - text: "k"
+            comment: "SDB key-value"
+          - text: "t"
+            comment: "Tiny ascii art"
+          - text: "v"
+            comment: "Interactive ascii art"
+          - text: "w [path]"
+            comment: "Write to path or display graph image (see graph.gv.format)"
+    subcommands:
+      - name: aga
+        summary: Data reference graph
+        cname: analysis_graph_dataref
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agA
+        summary: Global data references graph
+        cname: analysis_graph_dataref_global
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agc
+        summary: Function callgraph
+        cname: analysis_graph_callgraph_function
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agC
+        summary: Global callgraph
+        cname: analysis_graph_callgraph_global
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agd
+        summary: Diff graph
+        cname: analysis_graph_diff
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+          - name: addr
```

Comment:
what is this arg? try to explain it in the `summary` of this cmd as well.

Reply:
It's the address of another function which we will run the diff with.